### PR TITLE
FP-2915: Run lint-staged on npm-workflow

### PR DESCRIPTION
**Pull request by diasdm**
[FP-2915](https://movai.atlassian.net/browse/FP-2915)

Run lint-staged in CI:
* Lint-staged is a tool with similar capabilities to pre-commit
* We would like to use it to run linters on commit and also on the CI

[FP-2915]: https://movai.atlassian.net/browse/FP-2915?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by manuelnogueiramov** [comment]
Code wise looks good. 
I think we might benefit later from having a ticket associated with this configuration. What do you think?

**Review by manuelnogueiramov — APPROVED** [review]

File: .github/workflows/npm-workflow.yml [hunk 1]
```diff
@@ -120,6 +124,31 @@ jobs:
       if: ${{ failure() }}
       run: cat /github/home/.npm/_logs/*.log
 
+    - name: Run lint-staged
+      if: ${{ inputs.run_lint_staged == 'true' }}
+      run: |
+        # stash changes done by bumping version
+        git stash
+        echo ""
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/npm-workflow.yml]: what are those `echo ""` ?

**Review by AlexFernandes-MOVAI — APPROVED** [review]

File: .github/workflows/npm-workflow.yml (hunk 1 above)
Comment (diasdm) [reply .github/workflows/npm-workflow.yml]: Just there to help make the output more readable, see https://github.com/MOV-AI/frontend-npm-lib-scene/actions/runs/10593728006/job/29355866434#step:12:176

**Comment by diasdm** [comment]
> I think we might benefit later from having a ticket associated with this configuration. What do you think?

@manuelnogueiramov what do you mean? https://movai.atlassian.net/browse/FP-2915?

**Comment by manuelnogueiramov** [comment]
> > I think we might benefit later from having a ticket associated with this configuration. What do you think?
> 
> @manuelnogueiramov what do you mean? https://movai.atlassian.net/browse/FP-2915?

Oh sorry It wasn't in the title nor in the description so I couldn't find it and assumed there wasn't one.
As one of my team leaders used to say: "Assumption is the mother of all f*ups" :sweat_smile:

**Pull request merged**